text/profiles/0237-profiles-data-model.md

```diff
@@ -0,0 +1,1015 @@
+
+##### Field `scope_profiles`
+
+A list of ScopeProfiles that originate from a resource.
+
+##### Field `schema_url`
+
+This schema_url applies to the data in the "resource" field. It does not apply
+to the data in the "scope_profiles" field which have their own schema_url field.
+</details>
+
+#### Message `ScopeProfiles`
+
+A collection of Profiles produced by an InstrumentationScope.
+
+<details>
+<summary>Field Descriptions</summary>
+
+##### Field `scope`
+
+The instrumentation scope information for the profiles in this message.
+Semantically when InstrumentationScope isn't set, it is equivalent with
+an empty instrumentation scope name (unknown).
+
+##### Field `profiles`
+
+A list of Profiles that originate from an instrumentation scope.
+
+##### Field `schema_url`
+
+This schema_url applies to all profiles and profile events in the "profiles" field.
+</details>
+
+#### Message `Profile`
+
+A Profile represents a single profile generated by a profiler. It has an ID and it has a start time and end time. Profile contains lookup tables for Stacktraces, Mappings, Locations, Functions, Links, AttributeSets, and strings.
+
+Profile embeds one or more ProfileType messages — this allows to represent multiple profile types (e.g allocated objects and allocated bytes) in a single Profile message.
+
+<details>
+<summary>Field Descriptions</summary>
+
+##### Field `profile_id`
+
+A unique identifier for a profile. The ID is a 16-byte array. An ID with
+all zeroes is considered invalid. Profile ID can be used by other signals to uniquely identify a profile.
+This field is required.
+
+##### Field `start_time_unix_nano`
+
+start_time_unix_nano is the start time of the profile.
+Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+This field is semantically required and it is expected that end_time >= start_time.
+
+##### Field `end_time_unix_nano`
+
+end_time_unix_nano is the end time of the profile.
+Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+This field is semantically required and it is expected that end_time >= start_time.
+
+##### Field `attributes`
+
+attributes is a collection of key/value pairs. Attributes that are not specific to a particular profile (like server name) MUST be specified in Resource attributes instead.
+
+##### Field `dropped_attributes_count`
+
+dropped_attributes_count is the number of attributes that were discarded. Attributes
+can be discarded because their keys are too long or because there are too many
+attributes. If this value is 0, then no attributes were dropped.
+
+##### Field `original_payload`
+
+This is the original profile as retrieved from the profiler. For example, this can be a pprof or jfr encoded profile. The reason users might want to include these is because some formats are very generic and can not be easily converted to a more structured format.
+TODO: add a field that indicates the format of the original payload?
```

Comment:
I think for the `original_payload` field to be more useful there needs to be some indication of its format, otherwise the recipients won't know what to do with it or will have to infer it from other data or the end user will have to supply this knowledge.

However I am not sure how exactly the format may be indicated. There isn't a registry of profiling formats with something like a mime type assigned to each, is there?